runtime/caml/custom.h

```diff
@@ -75,6 +75,11 @@ extern struct custom_operations *
           caml_final_custom_operations(void (*fn)(value));
 
 extern void caml_init_custom_operations(void);
+
+extern struct custom_operations caml_nativeint_ops;
+extern struct custom_operations caml_int32_ops;
+extern struct custom_operations caml_int64_ops;
+extern struct custom_operations caml_ba_ops;
```

Comment:
I wondered why `extern` declarations and not a `CAMLextern`, given that the definitions are `CAMLexport`.  I guess that's because those 4 variables are not used directly by C stubs residing in DLLs.  Nonetheless, note that those 4 variables are referenced directly from ocamlopt-generated code.

Reply:
`extern` vs `CAMLextern` is inconsistent at the moment... the big PR will make sure that there's consistency